lisp/ess-r-package.el

```diff
@@ -472,6 +472,16 @@ disable the mode line entirely."
   :lighter ess-r-package-mode-line
   (if ess-r-package-mode
       (progn
+        ;; Forward relevant R settings for interacting with inferior
```

Comment:
Similar functionality is already there. See `ess-local-customize-alist` and `ess-with-current-buffer`. I would say we reuse some of this functionality. It might be worth creating a variable with a list of r-package specific variables used for transfer.

Reply:
Actually, you are not porting them from other buffers, but rather from ess-r-customize-alist directly. That's a bit different.